tests/test_s3_with_policy.tf

```diff
@@ -0,0 +1,47 @@
+module "s3_test_with_policy" {
+  source              = "../modules/s3"         # local testing by a developer
+  serviceregistry_api = var.serviceregistry_api # enter a service registry URI
+
+  bucket = "od-test-with-policy"
+  region = var.region
+
+  policy = replace(local.s3_public_read, "BUCKET", "od-test-with-policy")
```

Comment:
The policy document can be externalized as a template file.  
https://www.terraform.io/docs/providers/template/d/file.html

Reply:
Not a blocker

Reply:
DONE, thanks for the suggestion! It also helped to fix other problems like this.